### PR TITLE
added more debugging info for LQI/routingTable

### PR DESCRIPTION
**Pull request by TomTonic**
Sorry, I don‘t know TypeScript and I don‘t have a development environment.
Please regard this as a feature request that might be rather close to an actual implementation 😇

### CODE REVIEW

**Comment by TomTonic** [comment]
However, I‘m not sure if this will always yield in the same „with status code '1' ('FAILURE')“ in real wold scenarios. In this case, the „feature“ would be pretty useless.

File: src/adapter/z-stack/adapter/zStackAdapter.ts [hunk 1]
```diff
@@ -564,7 +564,8 @@ class ZStackAdapter extends Adapter {
                 );
                 const result = await response.start().promise;
                 if (result.payload.status !== ZnpCommandStatus.SUCCESS) {
-                    throw new Error(`LQI for '${networkAddress}' failed`);
+                    throw new Error(`LQI for '${networkAddress}' failed with status code '${
```
Comment (Koenkk) [inline src/adapter/z-stack/adapter/zStackAdapter.ts:567]: I think this is a good improvement, but could you change the format to:

```js
LQI for '${networkAddress}' failed with error: '${ZnpCommandStatus[result.payload.status]}' (${result.payload.status})
```

Comment (TomTonic) [reply src/adapter/z-stack/adapter/zStackAdapter.ts]: ok, done. I tried to add the same functionality to the ezsp adapter, yet the error code lookup did not work - so I skipped it for the ezsp adapter.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**